code/game/objects/structures/stool_bed_chair_nest/alien_nests.dm

```diff
@@ -19,14 +19,14 @@
 		for(var/buck in buckled_mobs) //breaking a nest releases all the buckled mobs, because the nest isn't holding them down anymore
 			var/mob/living/M = buck
 
-			if(user.get_int_organ(/obj/item/organ/internal/xenos/plasmavessel))
+			if(usr.get_int_organ(/obj/item/organ/internal/xenos/hivenode))
```

Comment:
please convert all of these back into `user`

Reply:
Donezo 👍